GAE/src/org/akvo/flow/rest/form/QuestionGroupDtoMapper.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2021 Stichting Akvo (Akvo Foundation)
+ *
+ * This file is part of Akvo Flow.
+ *
+ * Akvo Flow is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Akvo Flow is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Akvo Flow.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.akvo.flow.rest.form;
+
+import com.gallatinsystems.survey.domain.QuestionGroup;
+import com.google.appengine.api.datastore.KeyFactory;
+import java.util.List;
+import java.util.TreeMap;
+import javax.annotation.Nonnull;
+import org.waterforpeople.mapping.app.gwt.client.survey.QuestionGroupDto;
+import org.waterforpeople.mapping.app.gwt.client.survey.SurveyDto;
+
+public class QuestionGroupDtoMapper {
```

Comment:
Since we already have this one maybe we can merge them to avoid confusion? https://github.com/akvo/akvo-flow/blob/master/GAE/src/org/akvo/flow/domain/mapper/QuestionGroupDtoMapper.java